src/ell/stores/models/core.py

```diff
@@ -20,8 +17,16 @@
 from sqlmodel import Field, SQLModel, Relationship, JSON, Column
 from sqlalchemy import Index, func
 
-
 from typing import  Any
+import ell.types.serialize
+
+def utc_now() -> datetime:
```

Comment:
See if this lives elsewhere and maybe name current_timestamp or something less awkward